plugin/advancedsubscription/ajax/advsub.ajax.php

```diff
@@ -0,0 +1,248 @@
+<?php
```

Comment:
Use a complete words that describes the file:

advanced_subscription.ajax.php

instead of 

advsub.ajax.php